qtpy/QtGui.py

```diff
@@ -35,7 +48,16 @@
         QFontMetrics.width = lambda self, *args, **kwargs: self.horizontalAdvance(*args, **kwargs)
 elif PYSIDE6:
     from PySide6.QtGui import *
-    from PySide6.QtOpenGL import *
+
+    # Attempt to import QOpenGLWidget, but if that fails, don't throw an exception until the
+    # symbol is explicitly asked for
+    # this allows removing the sizeable OpenGL binaries from pyinstaller bundles, but
+    # maintains compatibility with apps that expect this symbol to be automatically imported
+    try:
+        from PySide6.QtOpenGLWidgets import QOpenGLWidget
+    except (ModuleNotFoundError, ImportError):
+        MISSING_OPENGL_NAMES['QOpenGLWidget'] = ('PySide6.QtOpenGLWidgets', 'pyopengl')
```

Comment:
The definition of the error message contents belongs closer to where the error happens.